src/sentry/migrations/0223_delete_old_sentry_docs_options.py

```diff
@@ -0,0 +1,466 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        for option in orm.Option.objects.filter(key__startswith='sentry:docs'):
```

Comment:
This could be improved with 1 query, and just do:

`orm.Option.objects.filter(key__startswith='sentry:docs').delete()`

Instead of the 36 queries that'd come from this (one for SELECT, 35 DELETE).